src/pke/examples/simple-ckks-bootstrapping.cpp

```diff
@@ -78,12 +76,12 @@ void SimpleBootstrapExample() {
     */
 #if NATIVEINT == 128
     ScalingTechnique rescaleTech = FIXEDAUTO;
-    usint dcrtBits               = 78;
-    usint firstMod               = 89;
+    uint32_t dcrtBits            = 78;
```

Comment:
is it possible to use the default param values here?
**src/pke/include/scheme/gen-cryptocontext-params-defaults.h:**
#if NATIVEINT == 128
        constexpr ScalingTechnique scalTech = FIXEDAUTO;
        constexpr uint32_t firstModSize     = 89;
        constexpr uint32_t scalingModSize   = 78;
#else
        constexpr ScalingTechnique scalTech = FLEXIBLEAUTOEXT;
        constexpr uint32_t firstModSize     = 60;
        constexpr uint32_t scalingModSize   = 50;
#endif